src/store/model/helpers/nvidia.ts

```diff
@@ -1,89 +1,73 @@
-import {Browser, Response} from 'puppeteer';
+import {Browser, Page, Response} from 'puppeteer';
 import {NvidiaRegionInfo, regionInfos} from '../nvidia-api';
 import {Config} from '../../../config';
 import {Link} from '../store';
 import {Logger} from '../../../logger';
 import open from 'open';
 import {timestampUrlParameter} from '../../timestamp-url-parameter';
 
-const nvidiaApiKey = '9485fa7b159e42edb08a83bde0d83dia';
-
 function getRegionInfo(): NvidiaRegionInfo {
 	const country = Array.from(regionInfos.keys()).includes(Config.store.country) ? Config.store.country : 'usa';
 
-	const defaultRegionInfo: NvidiaRegionInfo = {drLocale: 'en_us', fe2060SuperId: 5379432500, fe3080Id: 5438481700, fe3090Id: null, nvidiaLocale: 'en_us'};
+	const defaultRegionInfo: NvidiaRegionInfo = {currency: 'USD', drLocale: 'en_us', fe2060SuperId: 5379432500, fe3080Id: 5438481700, fe3090Id: null, nvidiaLocale: 'en_us'};
```

Comment:
May have been a copy paste error, but I think we do have a 3090 code here. You've got it down below at least! 😄 

```suggestion
	const defaultRegionInfo: NvidiaRegionInfo = {currency: 'USD', drLocale: 'en_us', fe2060SuperId: 5379432500, fe3080Id: 5438481700, fe3090Id: 5438481600, nvidiaLocale: 'en_us'};
```

Reply:
It is `null` in the existing `HEAD`. I've modified the function to remove duplication by checking if the regions map contains the country and falling back to usa if not.